engine/src/main/java/org/hibernate/validator/spi/properties/GetterPropertySelectionStrategy.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.spi.properties;
+
+import java.util.Set;
+
+import org.hibernate.validator.Incubating;
+import org.hibernate.validator.internal.properties.DefaultGetterPropertySelectionStrategy;
+
+/**
+ * <p>
```

Comment:
No `<p>` is needed here: it's the first sentence, `<p>` is just used to separate paragraphs.